concrete/src/Backup/ContentImporter/Importer/Routine/ImportSinglePageStructureRoutine.php

```diff
@@ -39,6 +41,10 @@ public function import(\SimpleXMLElement $sx)
                         $siteTree = $home->getSiteTreeObject();
                     }
 
+                    if (is_null($siteTree)) {
```

Comment:
`if ($siteTree === null) {` is much faster

Reply:
Now with extra speed ;)

Reply:
Lightning fast! :zap: